### ISSUE TITLE
can i init close the panel x_panel for default?

### ISSUE THREAD

**Issue by residente007** [ColorlibHQ/gentelella#917]
can i init close the panel x_panel for default?

I mean. Keep closed when load or reload or refresh the page

How can i do it?

**Comment by NevilleDoke** [comment]
Hii @residente007 you can add the below script to initially close the x_panel.

<script>
$(document).ready(function() {
    // Store the content of .x_content elements before removing them
    $(".x_panel").each(function() {
        var content = $(this).find(".x_content").detach();
        $(this).data("content", content);
    });
    $(".collapse-link").on("click", function() {
        var panel = $(this).closest(".x_panel"),
            icon = $(this).find("i"),
            content = panel.find(".x_content");

        if (content.length === 0) {
            // If .x_content does not exist, add it back
            content = panel.data("content");
            panel.append(content);
            content.hide(); // Hide the content initially to prepare for the slide toggle
        }

        // Toggle the .x_content visibility
        content.stop(true, true).slideToggle(200, function() {
            if (!panel.attr("style")) {
                panel.css("height", "auto");
            } else {
                panel.removeAttr("style");
            }
        });
        // Toggle the icon class
        icon.toggleClass("fa-chevron-down fa-chevron-up");
    });

    $(".close-link").click(function() {
        $(this).closest(".x_panel").remove();
    });
});
</script>

**Comment by NevilleDoke** [comment]
Hii @puikinsh please close this issue.

**Comment by puikinsh** [comment]
Hi 👋 — closing this as part of a triage of the long-tail issue queue.

Gentelella has gone through a complete modernisation since this issue was opened. The current release ([**v2.2.0**, April 2026](https://github.com/ColorlibHQ/gentelella/releases/tag/v2.2.0)) is essentially a different template under the same name:

- Bootstrap 5.3.8 + Vite 8 build system (was Bootstrap 4 + Gulp)
- jQuery completely removed — every script is vanilla JS modules
- DataTables 2.x with native Bootstrap 5 styling (was DataTables 1.x + jQuery)
- Sidebar fully rewritten with collapsed-state persistence across pages
- Unified card / panel design tokens (`--gt-radius-card`, `--gt-shadow-card`, …)
- Fresh dependency tree across the board (ESLint 10, Uppy 5, jsdom 29, ECharts tree-shaken −32 %, …)
- Full HTML validity sweep, security headers, normalised `<head>` across all pages

If you're still hitting this exact issue on the current version, please open a fresh issue with the page where it reproduces, your browser, and steps to reproduce — we'll take a fresh look.

If your original report was a framework-integration question (Django, Laravel, Angular, .NET, Vue, etc.), those are best asked in that framework's community — Gentelella is a static HTML/CSS/JS template, so wiring it into a backend is the same as any static asset bundle.

Thanks for the original report. 🙏

— Closed during the v2.2.0 housekeeping pass.
